src/org/loklak/harvester/strategy/PriorityKaizenHarvester.java

```diff
@@ -8,9 +8,9 @@
 
 public class PriorityKaizenHarvester extends KaizenHarvester {
 
-    private static class PriorityKaizenQueries extends KaizenQueries {
+    public static class PriorityKaizenQueries extends KaizenQueries {
 
-        private Comparator<ScoreWrapper> scoreComparator = (scoreWrapper, t1) -> (int) (scoreWrapper.score - t1.score);
+        private Comparator<ScoreWrapper> scoreComparator = (scoreWrapper, t1) -> (int) (1000 * (t1.score - scoreWrapper.score));
```

Comment:
Since `score` is always between 0 and 1, the difference is always between -1 and 1. So when we cast it to `int`, the comparison fails for two numbers whose difference is less.

For example, if the difference is 0.2, they appear to be same as `(int)0.2 == 0`. But if we scale it by 1000, it becomes a reasonable number after casting and comparator would work properly.